Create 18865

jasonatptld opened the pull request:
# Summary of Changes

- Please indicate the classes you are modifying:
 Added 18-865

# Checklist:

- [x] I have added my entries to the `README.md` table of contents in the root directory as links.

Make sure that your contributions:

- [x] Are accurate and will not mislead the reader
- [x] Do not encourage or cause academic integrity violations
- [x] Do not offend professors or unfairly criticize the university

jasonatptld committed:
Create 18865

jasonatptld commented:
I am not very experienced with Github.  I have no idea if I did this fork/pull request correctly.

mikinty commented on electives/18865:
```diff
@@ -0,0 +1,40 @@
+- Some students generally don’t like remote classes.  For some, something is lost when not having face to face interactions with the professor and other students.
+- Some students struggled with the “messy” nature of the lecture and lecture slides.
+- The texts that are listed in the syllabus are rarely used, and 99% of the class comes from lecture and lecture slides.
+-Instructor office hours are in Rwanda time.  This makes for limited access to the instructor outside of the few office hours/week. That said, the instructor does respond to WhatsApp and does try to make office hours align to Pittsburgh time zone.
```
Add space after `-`

mikinty commented on electives/18865:
```diff
@@ -0,0 +1,40 @@
+
+## Resources
+- https://www.mathworks.com/products/simulink.html
+
```
Extra space, get rid of this

mikinty commented on electives/18865:
```diff
@@ -0,0 +1,40 @@
+- The texts that are listed in the syllabus are rarely used, and 99% of the class comes from lecture and lecture slides.
+-Instructor office hours are in Rwanda time.  This makes for limited access to the instructor outside of the few office hours/week. That said, the instructor does respond to WhatsApp and does try to make office hours align to Pittsburgh time zone.
+
+## Resources
```
You should install a markdown formatter, it will do stuff like add an extra space after headers

mikinty commented on electives/18865:
```diff
@@ -0,0 +1,40 @@
+- Being comfortable with general AC power transmission and power electronics in general will make the class go a bit easier.  That said, being an ECE isn’t a requirement.  Due to the subject matter, there are often civil engineers taking the class.
+- As it is an online class, the lectures are usually recorded.  Accessing these proved invaluable when doing the homework as much of the homework comes from lecture.
+
+
```
Get rid of one of these spaces

mikinty commented on electives/18865:
```diff
@@ -0,0 +1,40 @@
+# 18-865: Power Electronics for Electric Utility Systems
```
Please make this an `.md` (Markdown) file. Right now it has no file extension

mikinty reviewed (changes requested):
This is pretty good, a few comments, and also you need to add this to the table of contents

jasonatptld committed:
Update 18865

Fixed white space.

jasonatptld committed:
Create 18865.md

Created new .md file for 18865.

jasonatptld committed:
Delete electives/18865

Deleted first version of 18865, which was not originally saved as a .md.

jasonatptld commented:
Updated white space, created a new .md file, deleted old non .md file.  I did previously create a link in the directory, but it may have been broken due to the non .md file.

jasonatptld commented:
I've become a bit lost with this process.  Is there anything else I need to do to get the 18-865 content added to the repository?

mikinty commented:
> I've become a bit lost with this process.  Is there anything else I need to do to get the 18-865 content added to the repository?

Yes same issues as mentioned before...you didn't modify the table of contents in this PR. You opened a new PR and I said to please put those changes in this one. You can see this PR only contains the new file, no changes to the table of contents.

jasonatptld committed:
Update README.md

Adding table of contents link for 18-865 (again).

mikinty reviewed (approved):
Thanks for addressing feedback!

jasonatptld commented:
OK.  I've added 18-865 to the table of contents (for the third time).  I'm not sure what happened to it the first time I added it.  Maybe I was in the wrong area of Github.  Please let me know if its all correct now.  Regards.
